lib/rainforest/cli/test_importer.rb

```diff
@@ -38,17 +38,22 @@ def threads
   end
 
   def export
-    tests = Rainforest::Test.all(page_size: 1000)
-    p = ProgressBar.create(title: 'Rows', total: tests.count, format: '%a %B %p%% %t')
-    Parallel.each(tests, in_threads: threads, finish: lambda { |_item, _i, _result| p.increment }) do |test|
+    test_ids = []
+    if @options.tests.length > 0
+      test_ids = @options.tests
+    else
+      test_ids = Rainforest::Test.all(page_size: 1000).map { |t| t.id }
+    end
```

Comment:
Let's do this the Ruby way :sunglasses:

You can set test_ids directly from the return value of your `if` block (so you don't have to set it to an empty array first and manually set it in the if block). Or you can do a ternary if it's short enough.

```ruby
foo = if false
  baz
else
  lol
end

foo == lol #true
```